packages/extension-sdk/src/connect/tile/tile_sdk.ts

```diff
@@ -53,58 +54,98 @@ export class TileSDKImpl implements TileSDK {
   }
 
   tileHostDataChanged(partialHostData: Partial<TileHostData>) {
-    this.tileHostData = { ...this.tileHostData, ...partialHostData }
+    // Ignore update messages if dashboard mounts not supported.
+    // Should never happen.
+    if (this.hostApi.isDashboardMountSupported) {
```

Comment:
seems to be a repeated pattern in this file. Does it make any sense to try to abstract?